examples/caa/caa.ipynb

```diff
@@ -0,0 +1,590 @@
+    "    # Note: you must have installed 'accelerate', 'bitsandbytes' to load in 8bit\n",
+    "    model = AutoModelForCausalLM.from_pretrained(\n",
+    "        model_name, use_auth_token=token, load_in_8bit=True\n",
+    "    )\n",
+    "    return model, tokenizer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/daniel/steering-vectors/.venv/lib/python3.10/site-packages/transformers/models/auto/tokenization_auto.py:711: FutureWarning: The `use_auth_token` argument is deprecated and will be removed in v5 of Transformers. Please use `token` instead.\n",
+      "  warnings.warn(\n",
+      "/home/daniel/steering-vectors/.venv/lib/python3.10/site-packages/transformers/models/auto/auto_factory.py:472: FutureWarning: The `use_auth_token` argument is deprecated and will be removed in v5 of Transformers. Please use `token` instead.\n",
+      "  warnings.warn(\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "923ce28b228e43fb81564d8a6ccd0fd7",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Downloading shards:   0%|          | 0/3 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "43d6989a0c0341559c33fbe59581ce73",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Loading checkpoint shards:   0%|          | 0/3 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "model_size = \"13b\" # or \"7b\"\n",
+    "model_name = f\"meta-llama/Llama-2-{model_size}-chat-hf\"\n",
+    "model, tokenizer = get_model_and_tokenizer(model_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set up Datasets"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "CAA requires data to be formatted in the style of Anthropic's Model-Written Evals. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import random\n",
+    "\n",
+    "# NOTE(dtch1997): I considered defining MWEData to be a dataclass instead\n",
+    "# which would make it clear what fields are expected and enable type-checking\n",
+    "# but I'm worried it will be confusing to the average researcher\n",
+    "# and I'd like to be consistent with Anthropic's MWE style\n",
+    "\n",
+    "MWEData = list[dict[str, str]]\n",
+    "\n",
+    "train_data: list[MWEData] = json.load(open(\"sycophancy_train.json\", 'r'))\n",
+    "test_data: list[MWEData] = json.load(open(\"sycophancy_test.json\", 'r'))\n",
```

Comment:
nit: rather than copying these files into our repo, could we instead download them from the CAA github repo in the notebook, and do whatever processing is necessary in the notebook? That will keep our repo clean and focused and also make it clear where this stuff came from.

I'm assuming this is the `generate_dataset.json` from CAA, so something like:
```
!wget https://raw.githubusercontent.com/nrimsky/CAA/main/preprocessed_data/generate_dataset.json
```

Reply:
done!